### PR TITLE
New scaler ActiveMQ Artemis docs

### PR DESCRIPTION
**Pull request by balchua**
Documentation on the new support for JMS based broker, ActiveMQ Artemis.

### CODE REVIEW

File: content/docs/1.5/scalers/artemis.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+```
+
+**Parameter list:**
+
+- `managementEndpoint`: "hostname:port" to connect to ActiveMQ Artemis management endpoint.
+- `queueName`: the name of the queue to check for the number of messages available.
+- `brokerName`: the name of the broker as defined in Artemis.
+- `brokerAddress`: the address name of the broker.
+- `queueLength` How much messages is in the queue. Default is 10. Optional.
+-
```
Comment (tomkerkhove) [inline content/docs/1.5/scalers/artemis.md:32]: Nit - We can remove this
```suggestion

```

File: content/docs/1.5/scalers/artemis.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
++++
+
+### Trigger Specification
+
+This specification describes the `artemis` trigger for ActiveMQ Artemis.
+
+```yaml
+triggers:
+- type: artemis
+  metadata:
```
Comment (tomkerkhove) [inline content/docs/1.5/scalers/artemis.md:17]: Can we configure username & password here as well similar to other scalers?

Those should only be pointers to env vars on the workload. We have this as a convenient method for those who don't want to use trigger auth.

Comment (balchua) [reply content/docs/1.5/scalers/artemis.md:17]: Ok sure.  I was just directing potential users to better practices. 😁

Comment (tomkerkhove) [reply content/docs/1.5/scalers/artemis.md:17]: Hehe I fully agree but we can only guide them and provide full docs 😅

**Comment by balchua** [comment]
creating a new PR. closing this one.

**Pull request closed without merging**